cnc_ctrl_v1/Report.cpp

```diff
@@ -236,6 +243,12 @@ void  returnError(){
         Serial.print(',');
         Serial.print(incSerialBuffer.spaceAvailable());
         Serial.println(F("]"));
+        float positionErrorLimit = 1.5;
```

Comment:
@blurfl Is the setting for adjusting the positionErrorLimit being applied? I was just messing with it and it doesn't seem to change the threshold when I change the setting, is this hard coded value overridden?